userguide/plugins.rst

```diff
@@ -248,6 +248,84 @@ are permanently deleted.
+
+Click
+:menuselection:`+ --> New Issue` .
+
+Set :guilabel:`Tracker` to *Feature*.
+
+.. the :guilabel:`Tracker` may change. Kris has said that he could add
+   another option in the drop down, related to plugin submisisons.
+
+* :guilabel:`Subject`:
+
+Contains the name and a description of the plugin.
+
+* :guilabel:`Description`:
+
+   * Find the correct port by searching for the plugin at
+     `<https://www.freshports.org/>`__. Link the URL of the port.
+   * Create and link the URL of the Git artifact repo for the plugin.
+     Refer to :numref:`table %s <Artifact Files>` for possible files
+     that are in the artifact repo.
+
+.. _Artifact Files:
+
+.. table:: Artifact Files
+   :class: longtable
+
+   +-------------------------+----------------------------------------------------------------------+
+   | File                    | Description                                                          |
+   +=========================+======================================================================+
+   | :file:`post_install.sh` | This script is run *inside* the jail after                           |
+   |                         | it has been created and packages installed.                          |
+   |                         | Enable services in /etc/rc.conf that need to start                   |
+   |                         | with the jail startup, apply configuration settings,                 |
+   |                         | and more.                                                            |
+   +-------------------------+----------------------------------------------------------------------+
+   | :file:`ui.json`         | Json file that accepts the  key or value options:                    |
+   |                         |                                                                      |
+   |                         | :samp:`adminportal: "http://%%IP%%/"`                                |
+   |                         |                                                                      |
+   |                         | The web-interface of the plugin for control and                      |
+   |                         | configuration .                                                      |
+   +-------------------------+----------------------------------------------------------------------+
+   | :file:`overlay/`        | Directory of files copied on top of the jail post-install.           |
+   |                         | For example, :file:`usr/local/bin/myfile` is placed in the jails     |
+   |                         | /usr/local/bin/myfile location. Can be used to supply custom         |
+   |                         | files and configuration data, scripts, and more.                     |
+   +-------------------------+----------------------------------------------------------------------+
+   | :file:`settings.json`   | Json file that controls plugins settings interface. The required     |
```

Comment:
JSON.
"plugins settings interface"?  Is that a possessive or a plural?  It's not clear, either way.  "controls" is sort of vague, too.